nominal/core/datasource.py

```diff
@@ -345,6 +374,23 @@ def _get_write_stream(
             track_metrics=True,
             max_workers=None,
         )
+    elif data_format == "rust_experimental":
+        try:
+            from nominal.experimental.rust_streaming.rust_write_stream import RustWriteStream
+        except ImportError as ex:
+            raise ImportError(
+                "nominal-streaming is required to use get_write_stream with data_format='rust_experimental'"
+            ) from ex
```

Comment:
minor nit but I think this error makes more sense to live inside `rust_write_stream.py`, that's more a source of truth of what lib its trying to import and why